packages/core/test/StringField_test.js

```diff
@@ -208,6 +208,21 @@ describe("StringField", () => {
       });
     });
 
+    it("should render ui:EmptyValue when formData is emptyString and ui:EmptyValue is defined", () => {
```

Comment:
This is the most important test in this PR. The `act(() => {})` gives the chance for the hook to run.